tunnel.go

```diff
@@ -211,6 +225,29 @@ func (api *API) UpdateTunnel(ctx context.Context, params TunnelUpdateParams) (Tu
 	return argoDetailsResponse.Result, nil
 }
 
+// UpdateTunnelConfiguration updates an existing tunnel for the account.
+//
+// API reference: https://api.cloudflare.com/#cloudflare-tunnel-configuration-properties
+func (api *API) UpdateTunnelConfiguration(ctx context.Context, params TunnelConfigurationParams) (Tunnel, error) {
+	if params.AccountID == "" {
+		return Tunnel{}, ErrMissingAccountID
+	}
+
```

Comment:
you'll also want to check `TunnelID` is provided too

Reply:
Done